rc-events-service/src/main/scala/com/epam/random_coffee/events/services/EventService.scala

```diff
@@ -0,0 +1,16 @@
+package com.epam.random_coffee.events.services
+
+import com.epam.random_coffee.events.model.Event
+
+import scala.concurrent.Future
+
+trait EventService {
+
+  def create(eventName: String): Future[Event]
+
+  def find(id: String): Future[Option[Event]]
```

Comment:
```suggestion
  def find(id: EventId): Future[Option[Event]]
```
Same for other methods here

Reply:
fixed